src/core/Suite.js

```diff
@@ -147,15 +147,18 @@ getJasmineRequireObj().Suite = function(j$) {
     return !args[0];
   }
 
-  function clone(obj) {
-    var clonedObj = {};
+  function extend(newObj, obj) {
     for (var prop in obj) {
       if (obj.hasOwnProperty(prop)) {
-        clonedObj[prop] = obj[prop];
+        newObj[prop] = obj[prop];
       }
     }
 
-    return clonedObj;
+    return newObj;
+  }
+
+  function clone(obj) {
```

Comment:
If the `clone` function is no longer used within Suite, we should just remove it.